app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  Checkbox,
+  ScrollView,
+  Flex,
+  Button,
+  Box,
+} from 'native-base'
+import StaRibbon from '../Components/StaRibbon'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([])
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) => {
+    return roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase())
+  })
+
+  return (
+    <>
+      <HStack marginTop={-10} marginBottom={12} alignItems={'flex-end'}>
+        <Heading size="md" fontWeight="bold" marginRight={-5}>
+          My Experience
+        </Heading>
+        <Flex flex={1} alignItems={'flex-end'}>
+          <StaThemeLogo />
+        </Flex>
+      </HStack>
+      <StaRibbon />
+      <ScrollView>
+        <FreeSearchBar
+          marginBottom="1"
+          marginTop="4"
+          handleSubmit={() => {
+            null
+          }}
+          handleChangeText={(text: string) => {
+            setSearchTxt(text)
+          }}
+        />
+        <VStack paddingTop="4" paddingBottom="2" margin="2">
+          <Checkbox.Group
+            marginLeft="2"
+            onChange={(value: string[]) => {
+              setSkillsValue(value)
+            }}
+            value={skillsValue}
+            accessibilityLabel="choose skills"
+          >
+            {filteredSkills.map((roleGroup: RoleGroup, index: number) => (
+              <Checkbox
+                borderColor="inputBorder.100"
+                _dark={{
+                  borderColor: 'white',
+                }}
+                value={roleGroup.groupName}
+                key={`${roleGroup.groupName} - ${index}`}
+              >
+                {roleGroup.groupName}
+              </Checkbox>
+            ))}
+          </Checkbox.Group>
+        </VStack>
+        <VStack marginX={6}>
+          <Button
```

Comment:
I'm getting an error `We can not support a function callback` when I select a checkbox. 

Also the checkbox turns blue when selected, think this should be purple on this page? Or just a tick if possible as per the design, but I suspect the NativeBase checkboxes don't behave like that out of the box.  

Also not sure what the desired behaviour is in terms of static content and content that scrolls (ie should the search box or the next buttons perhaps stay on screen when scrolling?)

Reply:
Hi Pamela, I will address other comments tomorrow but on the issue of the warning about a 'function callback ' I tried a number of fixes from googling about this issue. It seems to be an error that comes up in Nativebase from researching but haven't had any luck finding a fix yet. Will leave this until other issues are sorted. I'm terms of the buttons and search it's difficult to know if they are meant to remain fixed when scrolling as nothing on the designs to indicate but I presumed not.

Thanks for reviewing and will come back about the other comments...

Reply:
@PammyLo I have now made the searchbar fixed so that it remains in plave when the user scolls down. However i have not done this for the two buttons at the bottom of the screen as felt need more clarity from Mia on this. It makes sense  for the searchbar which is why i have made the suggestion you described above for this. As for the 'function callback' error i am no nearer resolving the issue. After research, it does appear a bug but with as yet no obvious fix for it. This was as recent as 3 months ago see https://github.com/GeekyAnts/NativeBase/issues/5098

Reply:
Wow that is annoying about that NativeBase warning -- ages without a fix for a very basic component (slightly terrifying!)

Could either use [a different checkbox](https://github.com/react-native-checkbox/react-native-checkbox) (but it might need to be styled) -- or we just ignore it?  I'm not sure whether it'll show up in Bugsnag (our bug logging service) or not as it's a warning.

Reply:
Makes sense to ask Mia what the intended behaviour is of the search and buttons. I don't know either, was more a question than a suggestion. 

I know the checkbox from Native Base is hard to change from solid to just a tick, possibly not a problem I'd say but also up to Mia or Joanna. Making it purple would be good though. 

That error really is annoying. I'll leave it up to you and Joe to decide how to deal with that ;)